content/messaging/agent-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -3803,6 +3806,11 @@ It returns the initial state of the current Agent.
 
 ### Set Away Status
 
+Sets an Agent's connection to the `away` state. You can use this method to manipulate the Agent's status. The method works per connection - all connections an Agents has (desktop, mobile, etc) must be in the `away` state. **Only then**, the Agent's status will change to `not_accepting_chats`.
```

Comment:
```suggestion
Sets an Agent's connection to the `away` state. You can use this method to manipulate the Agent's status. The method works per connection - all connections an Agent has (desktop, mobile, etc) must be in the `away` state for the Agent's status to be changed to `not_accepting_chats`.
```

Reply:
Hah, believe it or not, that was my first version. I changed it to avoid confusion by the "to be changed" structure, but maybe it looks like it should stay :)